pkg/conditions/conditions.go

```diff
@@ -0,0 +1,16 @@
+package conditions
+
+import (
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+)
+
+const (
+	// BindingReady indicates that the binding succeeded
+	BindingReady conditionsv1.ConditionType = "Binding is ready"
+
+	// BindingInProgress indicates that binding is in progress
+	BindingInProgress conditionsv1.ConditionType = "Binding is in progress"
```

Comment:
I'm unsure whether `InProgress` is required.

Whenever a manifest is created, the `Ready` condition is `Unknown`, and it is expected that something will be happening. After the reconcile function is finished, it should have a verdict regarding the condition status: it'll either be `True` or `False`.

Additionally, this information can be inferred by comparing the manifest's last update timestamp and the `Ready` condition's last transition timestamp; if the update timestamp is newer than the last transition, it should be considered in progress.

Another point here is that there are no asynchronous operations during the reconcile process, either everything works or not.